.github/workflows/main.yml

```diff
@@ -0,0 +1,57 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go: [ '1.14', '1.*' ]
```

Comment:
I think we agreed on using the version listed in go.mod (1.13 at the moment).

Reply:
Guess we adhere to the Go core supported versions:

> Each major Go release is supported until there are two newer major releases. For example, Go 1.5 was supported until the Go 1.7 release, and Go 1.6 was supported until the Go 1.8 release.

So I will update the go.mod instead of changing things here.

Reply:
Updated go.mod to use 1.14